packages/hoprd/src/api/v2.ts

```diff
@@ -11,6 +11,20 @@ import type { Application, Request } from 'express'
 import type Hopr from '@hoprnet/hopr-core'
 
 import type { LogStream } from './../logs'
+import { APIv2Settings } from './v2/logic/settings'
+
+export interface APIv2State {
+  aliases: Map<string, PeerId>
+  settings: APIv2Settings
+}
+
+const state: APIv2State = {
+  aliases: new Map<string, PeerId>(),
+  settings: {
+    includeRecipient: false,
+    strategy: "passive"
+  }
+}
```

Comment:
it looks like you stumbled upon another inconsistency in the protocol, it looks like we store `aliases` and `settings` not in the protocol level but in the app level (https://github.com/hoprnet/hoprnet/blob/731a29dd61ff81b678526992999737f2c31481d2/packages/hoprd/src/commands/index.ts#L31-L34).
For now, let's keep it like this until I discuss with tech team on what the architecture should look like.

Reply:
fixed